src/containers/AccountConnection.jsx

```diff
@@ -138,11 +138,30 @@ class AccountConnection extends Component {
       messages.push({message: 'account config details', params: {folder: account.folderPath}})
     }
 
+    if ([
+      'orange',
+      'orangelivebox',
+      'orangemobile'
+    ].indexOf(account.account_type) !== -1) {
+      messages.push(
+        {message: 'account config delay', params: {delayInDays: 15}}
+      )
+    }
     this.handleSuccess(account, messages)
   }
 
   handleUpdateSuccess (account) {
-    this.handleSuccess(account, [{message: 'account update success'}])
+    const messages = [{message: 'account update success'}]
+    if ([
+      'orange',
+      'orangelivebox',
+      'orangemobile'
+    ].indexOf(account.account_type) !== -1) {
+      messages.push(
+        {message: 'account config delay', params: {delayInDays: 15}}
+      )
+    }
+    this.handleSuccess(account, messages)
```

Comment:
I know, but it seems a bit confusing and not interesting to create a separate code just for that here.